metadata-schema/closureSchemaClosed.schema.json

```diff
@@ -0,0 +1,88 @@
+{
+  "$id": "/schema/closed-closure",
+  "type": "object",
+  "allOf": [
+    {
+      "if": {
+        "properties": {
+          "closure_type": {
+            "enum": ["Closed", "closed", "CLOSED"]
+          }
+        }
+      },
+      "then": {
+        "properties": {
+          "closure_start_date": {
```

Comment:
from the Jira card  - the schema requirements are minimal 
for closure_start_date
type:string   - format not required 
it is simpler to handle the error messages if there is only one possible error. If value not set we will have converted to null so there will be a type error. Format errors will come from the base schema

Closed Schema Requirements 
Validate that the following fields are not null:
closure_start_date (type = string)
closure_period (type = integer)
foi_exemption_code (type = array)
foi_exemption_asserted (type = string)
title_closed (type = boolean)
description_closed (type = boolean)

Validate that IF title_closed is Yes THEN
title_alternate is not null (type = string)

Validate that IF description_closed is Yes THEN
description_alternate is not null (type = string)